main.py

```
@@ -0,0 +1,250 @@
+    ]
+
+    if relevant_services:
+        print(f"\n🔍 Valid services with ws ports [{len(relevant_services)}]:")
+        for name, ns, port, node_type in relevant_services:
+            print(f"– {name}")
+    else:
+        print("⚠️ No valid services found with ws ports.")
+    return relevant_services
+
+def fetch_open_bu_incidents():
+    url = getattr(cfg, "BETTER_UPTIME_URL")
```

Ruff (0.11.9)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)